.travis.yml

```diff
@@ -19,6 +19,7 @@ install:
 - pip install pytest
 - pip install flake8
 - pip install pylint
+- pip install ufodiff
```

Comment:
As we get an ERROR on the `com_daltonmaag_check_ufolint` test on Travis, it probably means we need to install ufolint here as well.

Reply:
https://travis-ci.org/googlefonts/fontbakery/builds/357045774

![screenshot at 2018-03-22 16 55 57](https://user-images.githubusercontent.com/213676/37795216-9fc17ce0-2df2-11e8-8273-c49401f3b262.png)